napari/components/_dims/view.py

```diff
@@ -170,10 +170,11 @@ def _trim_sliders(self, number_of_sliders):
         ----------
         number_of_sliders : new number of sliders
         """
-        for slider_num in range(self.nsliders, number_of_sliders):
+        for slider_num in range(number_of_sliders + 1):
```

Comment:
Can you explain this? I don't understand why there is a +1 here...

Reply:
this has now been fixed, comment has been added too